src/GameManipulator.php

```diff
@@ -254,108 +277,82 @@ public function isShipSunk(Ship $ship, Game $game): bool
      * Get opponent ships that have been sunk
      * 
      * @param Game $game 
+     * @param Player $player
      *  
      * @return array
      */
-    public function getOpponentShipsSunk(Game $game): array {
+    public function getOpponentShipsSunk(Game $game, Player $player): array
```

Comment:
Better! But i'm still confused about which player it is? The current one? The opponent?